### PR TITLE
(maint) Upgrade jruby 9.3.11

### PR DESCRIPTION
**Pull request by justinstoller**
Please add all notable changes to the "Unreleased" section of the CHANGELOG.

### CODE REVIEW

**Review by steveax — APPROVED** [review]

**Comment by justinstoller** [comment]
I've tested this and I think it's good to go, but going to wait for CI issues / release shenanigans to calm down before merging.

**Comment by justinstoller** [comment]
CI shenanigans have calmed down. Rebased onto HEAD of the 5.x branch and updated the changelog.

**Pull request merged**
